Tests/images/custom_gimp_palette.gpl

```diff
@@ -1,12 +1,12 @@
 GIMP Palette
 Name: custompalette
 Columns: 4
-#
-  0   0   0     Index 3
- 65  38  30     Index 4
-103  62  49     Index 6
- 79  73  72     Index 7
-114 101  97     Index 8
-208 127 100     Index 9
-151 144 142     Index 10
-221 207 199     Index 11
+# Original written by David Wetz in https://stackoverflow.com/questions/815836/im-creating-a-program-that-generates-a-palette-from-a-true-color-image-need-hel/815855#815855
```

Comment:
Just an explanation of what is happening here - this isn't a random attribution. You're creating a comment that is over 100 characters.

Reply:
comments with >100 char lines long should not "break" the file. Even if we are reformatting here, the URL alone is wider than 100 chars in this case, so even breaking this in a functional way would  still have lines wider than that.